drv/cosmo-seq-server/src/main.rs

```diff
@@ -633,6 +647,16 @@ impl ServerImpl {
             notifications::SEQ_IRQ_MASK,
             sys_api::IrqControl::Enable,
         );
+        // Enable the undervoltage warning PMBus alert from the Vcore
+        // regulators.
+        //
+        // Yes, we just ignore the error here --- while that seems a bit
+        // sketchy, but what else can we do? It seems pretty bad to panic and
+        // say "nope, the computer won't turn on" because we weren't able to do
+        // an I2C transaction to turn on an interrupt that we only use for
+        // monitoring for faults. The initialize method will retry internally a
+        // few times, so we should power through any transient I2C messiness.
+        let _ = self.vcore.initialize_uv_warning();
```

Comment:
Should we log an error here?  It looks like `VCore::initialize_uv_warning` doesn't log errors internally.

Reply:
`initialize_uv_warning` actually _does_ log errors in the `vcore` ringbuf internally, since it calls into `retry_i2c_txn`:
https://github.com/oxidecomputer/hubris/blob/31555d3adcc3ef19b72e95396f6b53be0a335d84/drv/cosmo-seq-server/src/vcore.rs#L126-L132
and `retry_i2c_txn` will ringbuf any errors along with which regulator they came from and what it was trying to do:
https://github.com/oxidecomputer/hubris/blob/31555d3adcc3ef19b72e95396f6b53be0a335d84/drv/cosmo-seq-server/src/vcore.rs#L431-L432